src/ucp/core/ucp_ep.c

```diff
@@ -57,6 +57,9 @@ static ucs_stats_class_t ucp_ep_stats_class = {
 };
 #endif
 
+ucp_tl_bitmap_t ucp_tl_bitmap_max = {{UINT64_MAX, UINT64_MAX}};
+ucp_tl_bitmap_t ucp_tl_bitmap_min = {{0, 0}};
```

Comment:
I added an initializer for an "all 0" bitmap. It's impossible to have an "all 1" initializer of the same form (i.e `bitmap ucp_tl_bitmap_t ucp_tl_bitmap_max = UCS_BITMAP_INIT_MAX;'`) using the C syntax.

Our options are:
1. Leave it as it.
2. Add a separate macro i.e `ucp_tl_bitmap_t ucp_tl_bitmap_max; UCS_BITMAP_INIT_MAX(ucp_tl_bitmap_max)`.
3. Add a macro/func which would *return* a bitmap, but it would require passing its length i.e `ucp_tl_bitmap_t ucp_tl_bitmap_max = UCS_BITMAP_MAX(128);`.

I'm in favor of (1). What do you say?

Reply:
ok, but let's move those definitions to ucp_types.h and make them const